site/it/community/contribute/code_style.md

```diff
@@ -0,0 +1,164 @@
+Fino ad allora, tutto il codice bisognerà che sia compatibile con le versioni di Python elencate sopra.
+
+Queste linee dovranno essere presenti in ogni file Python:
+
+
+```python
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+```
+
+Usate `six` per scrivere codice compatibile (per esempio, `six.moves.range`).
+
+
+## Style di codifica C++
+
+Le modifiche al codice C++ di TensorFlow dovrebbero essere conformi alla 
+[Guida Google di Stile C++](https://google.github.io/styleguide/cppguide.html). 
+Usare `clang-tidy` per controllare le vostre modifiche al codice C/C++.
+
+Per installare  `clang-tidy` su Ubuntu 16+:
+
+
+```bash
+$ apt-get install -y clang-tidy
+```
+
+Potete controllare un file C/C++ con il seguente:
+
+```bash
+$ clang-format <my_cc_file> --style=google > /tmp/my_cc_file.cc
+$ diff <my_cc_file> /tmp/my_cc_file.cc
+```
+
+## Altri linguaggi
+
+*   [Guida di Stile Google per Java](https://google.github.io/styleguide/javaguide.html)
+*   [Guida di Stile Google per JavaScript](https://google.github.io/styleguide/jsguide.html)
+*   [Guida di Stile Google per Shell](https://google.github.io/styleguide/shell.xml)
+*   [Guida di Stile Google per Objective-C](https://google.github.io/styleguide/objcguide.html)
+
+
+
+
+## Convenzioni ed usi speciali in TensorFlow
+
+### Tensori
+
+*   Le operazioni che hanno a che trattano batch possono ipotizzare che la **prima dimensione** di
+    un Tensore sia la dimensione del batch.
+*   In molti modelli, l'**ultima dimensione** è il numero di _canali_.
+*   Le dsimensioni escluse la prima e l'ultima, di solito, rappresentano le dimensioni dell _spazio_
+    : la lunghezza della sequenza, o la dimensione dell'immagine.
+    *   Quando possibile, usate un operatore sovraccarivcato tra Tensori, invece di una funzione di TensorFlow. 
+    Per esempio, noi preferiamo `**`, `+`, `/`, `*`, `-`, `and/or` a
+    `tf.pow`, `tf.add`, `tf.divide`, `tf.multiply`, `tf.subtract`, e `tf.logical_*` —
+    a meno che sia richiesto uno specifico nome per l'operazione.
+
+
+### Operazioni Python
+
+Un' _operazione Python_ è una funzione che, dati in input tensori e parametri,
+crea una parte del grafo, e ritorna come risultato dei tensori.
+*   Il primo argomento dovrà essere un tensore, seguito dai parametri Python di base.
+    L'ultimo argomento è `name` con valore di default `None`.
+*   Gli argomenti TensorFlow dovrebbero essere un tensore singolo o un elenco "iterabile" di 
+    tensori. Perché, "un Tensore o una lista di Tensori" è troppo generico. Vedere `assert_proper_iterable`.
+*   Le opreazioni che prendono tensori come argomenti ed usano operazioni C++, 
+    dovranno chiamare `convert_to_tensor` per convertire input non-tensori in tensori.
+    Notare che gli argomenti sono acora descritti nella documentazione come oggetti 
```

Comment:
In general I would say "Da notare" instead of "notare", but just "notare" sounds ok too.
What do you think?

Reply:
Also "Nota" could be ok

Reply:
Other verbs in in the same section are in infinite form ("... chiamare", "... essere", "... chiamare", "... passare" ). I used "Notare" to maintain the same form. In my opinion "Nota" should be a sort of "brake" in "consecutio temporum". In any case, I'm available for a final suggestion...